roles/openshift_logging/defaults/main.yml

```diff
@@ -97,7 +97,7 @@ openshift_logging_es_nodeselector: "{{ openshift_hosted_logging_elasticsearch_no
 # openshift_logging_es_config is a hash to be merged into the defaults for the elasticsearch.yaml
 openshift_logging_es_config: {}
 openshift_logging_es_number_of_shards: 1
-openshift_logging_es_number_of_replicas: 0
+openshift_logging_es_number_of_replicas: 1
```

Comment:
Should stay 0 for same reason -- this is specific to index shard replicas not pods